datasets/hebrew_projectbenyehuda/README.md

```diff
@@ -0,0 +1,190 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- he
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- parsing
```

Comment:
is it really a dataset for parsing ?

Reply:
Good question, I think It's not. The dataset is like the project dataset by Project Gutenberg. Removed it. What is the best option for task_categories except other? Tried to search for something like this at the project but pg19 doesn't have a dataset card

Reply:
You can replace `other` by `sequence-modeling` to end up with:
```
task_categories:
- sequence-modeling
task_ids:
- language-modeling
```